src/Avalonia.Controls.ColorPicker/Avalonia.Controls.ColorPicker.csproj

```diff
@@ -0,0 +1,21 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <TargetFrameworks>net6.0;netstandard2.0</TargetFrameworks>
+    <!--<PackageId>Avalonia.Controls.ColorPicker</PackageId>-->
```

Comment:
@maxkatz6 I am unable to build the project or solution. The error I get is similar to "...\Avalonia\src\Avalonia.Controls.ColorPicker\Avalonia.Controls.ColorPicker.csproj : error NU1101: Unable to find package Avalonia.Controls.ColorPicker. No packages exist with this id in source(s): api.nuget.org, C:\Program Files\dotnet\library-packs, dotnet-eng, skiasharp"

I don't understand how Nuget is configured but it appears someone needs to add the package to api.nuget.org to continue and then this line uncommented.

Reply:
You don't really add package to the nuget, you only can upload already packed one.

Reply:
I think this error might come from ApiDiff tool, which downloads previously released stable package to compare with local. Try to remove ```<Import Project="..\..\build\ApiDiff.props" />``` import, or comment. It will make sense only after this package released at least once.

Reply:
Yes, commenting this out resolved that error. Good catch!